editor/components/post-schedule/index.js

```diff
@@ -13,41 +11,21 @@ import { settings } from '@wordpress/date';
 /**
  * Internal dependencies
  */
-import './style.scss';
-import PostScheduleClock from './clock';
+import { DateTimePicker } from '@wordpress/components';
 import { getEditedPostAttribute } from '../../store/selectors';
 import { editPost } from '../../store/actions';
 
 export function PostSchedule( { date, onUpdateDate } ) {
-	const momentDate = date ? moment( date ) : moment();
 	const handleChange = ( newDate ) => {
 		onUpdateDate( newDate.format( 'YYYY-MM-DDTHH:mm:ss' ) );
 	};
 
-		// To know if the current timezone is a 12 hour time with look for "a" in the time format
-		// We also make sure this a is not escaped by a "/"
-	const is12HourTime = /a(?!\\)/i.test(
-		settings.formats.time
-			.toLowerCase() // Test only the lower case a
-			.replace( /\\\\/g, '' ) // Replace "//" with empty strings
-			.split( '' ).reverse().join( '' ) // Reverse the string and test for "a" not followed by a slash
-	);
-
-	return [
-		<DatePicker
-			key="date-picker"
-			inline
-			selected={ momentDate }
-			onChange={ handleChange }
-			locale={ settings.l10n.locale }
-		/>,
-		<PostScheduleClock
-			key="clock"
-			selected={ momentDate }
-			onChange={ handleChange }
-			is12Hour={ is12HourTime }
-		/>,
-	];
+	return <DateTimePicker
```

Comment:
In general, for multi-lines elements, we tend to wrap them in `()` so something like 

```js
return (
  <Component
    prop={ value }
  />
);
```